docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -280,24 +283,38 @@ To use GPU acceleration on Linux, several environment variables are required or
 
 ### Known issues
 
-#### 1. Ubuntu 22.04 and Linux kernel 6.2.0 may cause performance bad (driver version < stable_775_20_20231219)
+#### 1. Potential suboptimal performance with Linux kernel 6.2.0
 
-For driver version < stable_775_20_20231219, the performance on Linux kernel 6.2.0 is worse than Linux kernel 5.19.0. You can use `sudo apt update && sudo apt install -y intel-i915-dkms intel-fw-gpu` to install the latest driver to solve this issue (need reboot OS).
+For Ubuntu 22.04 and driver version < stable_775_20_20231219, the performance on Linux kernel 6.2.0 is worse than Linux kernel 5.19.0. You can use `sudo apt update && sudo apt install -y intel-i915-dkms intel-fw-gpu` to install the latest driver to solve this issue (need to reboot OS).
 
 Tips: You can use `sudo apt list --installed | grep intel-i915-dkms` to check your intel-i915-dkms's version, the version should be latest and >= `1.23.9.11.231003.15+i19-1`.
 
-#### 2. Driver installation meet unmet dependencies: intel-i915-dkms
+#### 2. Driver installation unmet dependencies error: intel-i915-dkms
 
-The last apt install command of the driver installation may get following error:
+The last apt install command of the driver installation may produce the following error:
 
 ```
 The following packages have unmet dependencies:
  intel-i915-dkms : Conflicts: intel-platform-cse-dkms
                    Conflicts: intel-platform-vsec-dkms
 ```
 
-You can use `sudo apt install -y intel-i915-dkms intel-fw-gpu` to instead. As the intel-platform-cse-dkms and intel-platform-vsec-dkms are already provided by intel-i915-dkms.
+You can use `sudo apt install -y intel-i915-dkms intel-fw-gpu` to install instead. As the intel-platform-cse-dkms and intel-platform-vsec-dkms are already provided by intel-i915-dkms.
 
 ### Troubleshooting
 
-todo
+#### 1. Cannot open shared object file: No such file or directory
+
+Error where libmkl file is not found, for example,
+
+```
+OSError: libmkl_intel_lp64.so.2: cannot open shared object file: No such file or directory
+```
+```
+Error: libmkl_sycl_blas.so.4: cannot open shared object file: No such file or directory
+```
+
+The reason for such errors is that oneAPI has not been initialized properly before running BigDL-LLM code or before importing IPEX package.
+
+* Step 1: Make sure to execute setvars.sh of oneAPI Base Toolkit before running BigDL-LLM code.
+* Step 2: Make sure to install matching versions of BigDL-LLM/pytorch/IPEX and oneAPI Base Toolkit. BigDL-LLM with Pytorch 2.1 should be used with oneAPI Base Toolkit version 2024.x.x. BigDL-LLM with Pytorch 2.0 should be used with oneAPI Base Toolkit version 2023.x.x.
```

Comment:
to -> you
do not use x.x? Just use 2024.0 and 2023.2? @qiuxin2012 @Oscilloscope98

Reply:
Ok. We can update the docs when there are new minor versions.

Reply:
Yes I think 2024.0 and 2023.2 are ok